app/import.go

```diff
@@ -19,7 +18,7 @@ import (
 
 func stopOnError(err LineImportWorkerError) bool {
 	if err.Error.Id == "api.file.upload_file.large_image.app_error" {
-		mlog.Warn(fmt.Sprintf("Large image import error: %s", err.Error.Error()))
+		mlog.Warn("Large image import error", mlog.String("app_error", err.Error.Error()))
```

Comment:
Probably here makes more sense to use `mlog.Err(err.Errror)`

Reply:
My bad, I should have through through. Thanks for the suggestion!